lib/monitor.js

```diff
@@ -121,10 +121,15 @@ internals.Monitor.prototype.start = function (callback) {
         };
     };
 
+    var reporterIndex = 0;
+
     Items.serial(this.settings.reporters, function (item, next) {
 
         var reporter;
 
+        // Use hint of reporter index
+        var errorHint = '(' + reporterIndex++ + ')';
```

Comment:
Can you split into two lines? This is hard to read there are 4 plus signs very close to each other.